CMakeLists.txt

```diff
@@ -43,10 +43,13 @@ elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
     #add_definitions( -Wno-class-memaccess )
     add_definitions( -Wno-unused-private-field )
     add_definitions( -Wno-exceptions )
+    add_definitions( -shared ) 
+    add_definitions( -fPIC )
 
 # Clang
 elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
-
+    add_definitions( -shared ) 
```

Comment:
whats this?

Reply:
-shared : creates a shared object for imgui